lib/pages/chat_details/chat_details_edit.dart

```diff
@@ -90,7 +93,11 @@ class ChatDetailsEditController extends State<ChatDetailsEdit>
   ) {
     final listAction = [
       EditChatAvatarContextMenuActions.edit,
-      EditChatAvatarContextMenuActions.delete,
+      if ((room?.avatar != null ||
+              avatarFilePicker != null ||
+              avatarAssetEntity != null) &&
+          !_isDeleteAvatar)
```

Comment:
could you define a variable for this pls

Reply:
A method

Reply:
Done